client/src/components/register-page/register-form/form/BoxElement.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Button, Typography } from '@mui/material'
+import headerLogo from "/src/images/headerLogo.png"
+import "../../../Styles/LoginRegisterStyles.scss"
+import { UserInput } from '../user-input/UserInput'
+import { SignUpLogInBtn } from '../../../sign-in__sign-up-Btns/SignUp-LogIn-Btn'
```

Comment:
Strange folder name 'sign-in__sign-up-Btns'. It can be more shorter and explanatory.